open-apparel-registry/open-apparel-registry
# Update logo and copy

## Overview

Updates the logo, powered-by footer image, newsletter signup copy, and facility status report copy. 

Connects #1432, #1438, #1439

## Demo

<img width="1416" alt="Screen Shot 2021-08-04 at 3 16 42 PM" src="https://user-images.githubusercontent.com/21046714/128243927-2b1763ce-2e2e-4dd9-b25e-d04e0940a3ae.png">
<img width="833" alt="Screen Shot 2021-08-04 at 3 19 40 PM" src="https://user-images.githubusercontent.com/21046714/128243930-459038a2-45a6-4836-a6c3-5c4f6bb296b7.png">
<img width="632" alt="Screen Shot 2021-08-04 at 3 20 12 PM" src="https://user-images.githubusercontent.com/21046714/128243936-40b8f006-3ff2-4dcc-a63a-4032d6022169.png">
<img width="750" alt="Screen Shot 2021-08-04 at 3 20 49 PM" src="https://user-images.githubusercontent.com/21046714/128243939-c3ff0a73-3a13-4f21-93a6-a604cd949e37.png">

## Testing Instructions

* Run `./scripts/server`
* Navigate to [embedded map](http://localhost:6543/?embed=1&contributors=3) (replace the contributor with a contributor with an embed config) and view the footer image. 
* Navigate to the full website and view the logo in the header
* Navigate to a facility and click 'report facility as closed'. Check the text. 
* Log out and select 'register'. Check the newsletter signup text. 

## Checklist
